vllm/model_executor/models/minicpm3.py

```diff
@@ -241,6 +241,23 @@ class MiniCPM3ForCausalLM(MiniCPMForCausalLM):
     # `embedding_modules` and `embedding_padding_modules`
     # are inherited from MiniCPMForCausalLM
 
+    # BitandBytes specific attributes
+    default_bitsandbytes_target_modules = [
+        ".q_a_proj.",
+        ".q_b_proj.",
+        ".kv_a_proj_with_mqa.",
+        ".kv_b_proj.",
+        ".o_proj.",
+        ".gate_proj.",
+        ".up_proj.",
+        ".down_proj.",
+    ]
```

Comment:
```suggestion

```
We don't need `default_bitsandbytes_target_modules` anymore once #10597 merged

Reply:
I updated~ Please take a look. Thank you!